pa11ycrawler/pipelines.py

```diff
@@ -182,6 +221,7 @@ def process_item(self, item, spider):
                 )
             )
 
+        self.check_title_match(item['page_title'], stdout, spider.logger)
```

Comment:
If all we do is log a message, it'll still write results. If we raise an exception, I would _assume_ that Scrapy will drop this item from the pipeline and not write results for the page, so it will appear that the page was never crawled. I would further assume that raising an exception would not actually cause the crawler to stop crawling -- I believe I've done that a few times while developing this code, and it was always able to keep chugging along. However, it's probably safer to just log rather than raise, if we want to be sure that this will keep running.